node/src/data/ledger.rs

```diff
@@ -15,7 +15,7 @@ pub struct Transaction {
 
 impl Transaction {
     pub fn hash(&self) -> [u8; 32] {
-        self.inner.hash().to_bytes().into()
+        self.inner.hash().to_bytes()
```

Comment:
Ledger.rs is deleted in `node-data`-related PR and this clippy warning was addressed too.

Reply:
You're right, I'm going to rebase this PR on top of your changes